client/state/sites/blog-stickers/test/reducer.js

```diff
@@ -89,5 +93,29 @@ describe( 'reducer', () => {
 				),
 			).to.deep.equal( { 123: [ 'dont-recommend' ] } );
 		} );
+
+		it( 'should remove a sticker', () => {
+			expect(
+				items(
+					{ 123: [ 'dont-recommend' ] },
+					{
+						type: SITES_BLOG_STICKER_REMOVE,
+						payload: { blogId: 123, stickerName: 'dont-recommend' },
+					},
+				),
+			).to.deep.equal( { 123: [] } );
+		} );
+
+		it( 'should not remove any stickers if the blog does not have that sticker', () => {
+			expect(
+				items(
+					{ 123: [ 'okapi-friendly' ] },
```

Comment:
ditto on the deep freeze, though you'll have to pull this one to a const to do the strict equal later on